kubetest/util.go

```diff
@@ -63,7 +63,12 @@ func init() {
 
 // Returns $GOPATH/src/k8s.io/...
 func k8s(parts ...string) string {
-	p := []string{os.Getenv("GOPATH"), "src", "k8s.io"}
+	gp := os.Getenv("GOPATH")
+	if gp == "" {
+		return "Error: GOPATH not set, please check your golang env setting"
```

Comment:
okay, make the check returns both `error` and `string` then, don't mix them up, and let the caller check for error